package.json

```diff
@@ -1,95 +1,19 @@
 {
-  "name": "tinymce-ember",
-  "version": "3.0.2",
-  "description": "TinyMCE Ember addon",
-  "keywords": [
-    "ember-addon",
-    "tinymce",
-    "editor"
-  ],
-  "repository": "https://github.com/concordnow/tinymce-ember",
-  "license": "MIT",
-  "author": "Concord Worlwide Inc.",
-  "directories": {
-    "doc": "doc",
-    "test": "tests"
-  },
+  "private": true,
+  "workspaces": ["addon", "test-app"],
+
   "scripts": {
-    "build": "ember build --environment=production",
-    "lint": "npm-run-all --aggregate-output --continue-on-error --parallel \"lint:!(fix)\"",
-    "lint:fix": "npm-run-all --aggregate-output --continue-on-error --parallel lint:*:fix",
-    "lint:hbs": "ember-template-lint .",
-    "lint:hbs:fix": "ember-template-lint . --fix",
-    "lint:js": "eslint . --cache",
-    "lint:js:fix": "eslint . --fix",
+    "prepare": "cd addon && yarn build",
     "release": "release-it",
-    "start": "ember serve",
-    "test": "npm-run-all lint test:*",
-    "test:ember": "ember test",
-    "test:ember-compatibility": "ember try:each"
+    "test": "cd test-app && yarn test:ember"
   },
-  "dependencies": {
-    "ember-cli-babel": "^7.26.6",
-    "ember-cli-htmlbars": "^6.0.0",
-    "ember-modifier": "^3.0.0"
+  "volta": {
+    "node": "16.14.2",
+    "yarn": "1.22.17"
```

Comment:
Not yarn v3 ? (or v2 )

Reply:
It doesn't change anything except that it requires more config